pycontw/pycon.tw
# add TalkProposal export columns "cancelled"、"accepted"、"vote"

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
追加admin下的Proposal表，多匯出幾個字段
#779 加入兩個字段匯出"canceled" and "is_accepted"
#780 加入關聯review表下的vote字段匯出


## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'admin/proposals/talkproposal'
2. Click on 'list_display'
3. Click on to 'export'
4. See error
